modules/virt-preparing-nodes-and-cluster-pci.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc
+
+[id="virt-preparing-nodes-and-cluster-pci_{context}"]
+= Preparing nodes and the cluster
```

Comment:
@fabiand - The peer reviewer has noted that the paragraph under  **Preparing nodes and the cluster** should be moved into the About PCI passthrough section.

Our modular documentation adheres to 3 types of modules: Concept, Procedure, Reference. Any content we develop must fit into one of those three module types.

This is Modular Documentation Guide that explains each module and the purpose of the module:https://redhat-documentation.github.io/modular-docs/#creating-assemblies

In light of this feedback from the peer reviewer in the GitHub issue, the Preparing nodes and the cluster content needs to move under About PCI passthrough which is a Concept module.

Please advise if you have any feedback about this change. Thanks, Lavanya

Reply:
"Preparing nodes and the cluster" is part of a procedure, not a concept

Reply:
@fabiand I believe what @ousleyp has noted is that (my interpretation): the paragraph under "Preparing nodes and the cluster" needs to move into "About PCI passthrough" because it is summarizing what happens with the 3 procedures that follow. If it is an introduction for a single procedure (and that is not the case because it is summarizing what happens with 3 different procedures) that paragraph could have moved to precede that single procedure. @ousleyp - please feel free to dive into the conversation here.

Reply:
This has been resolved by rearranging content.